test/runtests.jl

```diff
@@ -1,9 +1,10 @@
 using ImageMagick
 using RegisterQD
 using Test
-
-include("util.jl")
-include("qd_random.jl")
-include("qd_standard.jl")
-include("gridsearch.jl")
-include("initial_tfm.jl")
+# 
+# include("util.jl")
+# include("qd_random.jl")
+# include("qd_standard.jl")
+# include("gridsearch.jl")
+# include("initial_tfm.jl")
+include("SD_calculator_test.jl")
```

Comment:
You could also add this to the `util.jl` tests.

While there are reasons to comment out test files, keep in mind you can also say `include("util.jl")` rather than `include("runtests.jl")`.

Reply:
Ooops. I was running tests from pkg> and forgot to put these tests back.